### ISSUE TITLE
Create a dashboard for displaying forms

### ISSUE THREAD

**Issue by coding-geek21** [QAZIMAAZARSHAD/Voters-Registration-Portal#250]
## Is your feature request related to a problem? Please describe.

When user logins , user can see a dashboard which shows forms which they want to access.

i would like to work on this please assign me @QAZIMAAZARSHAD 

## Additional context
Add any other context about the feature request here.

**Comment by nlok5923** [comment]
@coding-geek21 could you please elaborate how you will be proceeding with this issue like your implementation method, UI thoughts etc because it's quite a good feature to add up.

**Comment by coding-geek21** [comment]
> @coding-geek21 could you please elaborate how you will be proceeding with this issue like your implementation method, UI thoughts etc because it's quite a good feature to add up.

Yeah we can have cards and illustrations , on clicking the cards the user will get redirected to respective forms

**Comment by nlok5923** [comment]
@coding-geek21 it would be great if you can create a UI first and share it with us so that we can smoothly work on this feature.

**Comment by coding-geek21** [comment]
> @coding-geek21 it would be great if you can create a UI first and share it with us so that we can smoothly work on this feature.

Sure I will do 👍

**Comment by QAZIMAAZARSHAD** [comment]
@coding-geek21 any updates?

**Comment by coding-geek21** [comment]
Will make PR by tomorrow

**Comment by QAZIMAAZARSHAD** [comment]
> Will make PR by tomorrow

Fine

**Comment by QAZIMAAZARSHAD** [comment]
@coding-geek21 stuck somewhere?

It has been 27 days since you were assigned.

**Comment by coding-geek21** [comment]
Sorry I'm busy with college works. please assign it to someone
